supplementary_materials/notebooks/fastdraft-deepseek/fastdraft_deepseek.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    if not (model_dir / 'openvino_tokenizer.xml').exists():
</pre><p>I think there is no need in this line. In old notebooks we made additional conversion step for backward compatibility, because previous notebook versions does not have requirements to install openvino  tokenizers and users may have already preconverted models. As it is new notebook, tokenizer should be here after optimum-cli command</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2717/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Since we force here an older version of Optimum, the tokenizer is not converted by default, maybe there is a flag I can add to the `optimum-cli` command to convert the tokenizer?